pkg/health/health.go

```diff
@@ -61,9 +72,14 @@ func (h ConsulHealthChecker) WatchNodeService(nodename string, serviceID string,
 			})
 			if err != nil {
 				errCh <- err
+			}
+			kvCheck, err := h.consulStore.GetHealth(nodename, serviceID)
```

Comment:
It will be healthy if either check is present. There should not be an error if the health is not present for that node. GetHealth doesn't error on a nil response and consul's Get doesn't either.